src/test/java/com/google/devtools/build/lib/sandbox/cgroups/BUILD

```diff
@@ -0,0 +1,20 @@
+load("@rules_java//java:defs.bzl", "java_test")
+
+java_test(
```

Comment:
@bigelephant29 quite a lot of tests don't seem to have targets
(Nothing showed up via `bazel query` or I couldn't trigger the test to fail with a faulty value)

I've added MemoryTest to start which had some use of `memory.max`.

Reply:
Thanks for flagging this! This is indeed an internal issue.

I'm working on a fix to export this BUILD file. I'll let you know once the fix is shipped. Please wait a little bit.

Reply:
Please rebase onto 504355262d0045435eea3d7b9077c82bd1e581f0! Thank you 😄

Reply:
Done!